### DISCUSSION TITLE
Can't access login page after Softaculous install

### DISCUSSION THREAD

**Discussion by aleyoscar** [firefly-iii/firefly-iii#6019] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [X] ... it's [definitely me, not you](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

After installing version 5.7.2 through Softaculous, I am unable to access the main login page at /public/login in order to start the initial setup process.

### Expected behaviour

Navigating to my site, whether it be https://example.com, https://example.com/public, or https://example.com/public/login, all throw the same error in both chromium browsers and firefox.

### Debug information

I can't access the debug page, but there is an error_log file in the root directory which shows the following:

```
[21-Apr-2022 18:48:24 UTC] PHP Fatal error:  Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 8.0.2". You are running 7.4.28. in /home/user/firefly/vendor/composer/platform_check.php on line 24
```

### Additional info

I have a phpinfo script in the root directory which shows the PHP version on my server is 8.0.17, so I'm not sure why it throws a PHP error. In chromium browsers I get an ERR_HTTP2_PROTOCOL_ERROR when trying to navigate to the page, and in firefox I get a "Secure Connection Failed" error. I have SSL certificates using Lets Encrypt, and without the Firefly III installation it is working fine (I just drop in a simple index.html file and it displays fine, also when opening the php script AFTER installation at https://example.com/phpinfo.php I get no security or certificate errors)

Since I am using a shared hosting provider my only option is to use Softaculous, which is stuck at Firefly III v5.7.2. I have no idea where to start troubleshooting this, any ideas?

**Comment by JC5** [comment]
There is a chance that your command line php version is different from the Apache version. In that case, it'll never be able to install correctly.

**Comment by aleyoscar** [comment]
It seems my hosting provider can't upgrade PHP to it's proper version for firefly-iii, so they are removing it from their Softaculous packages. Looks like I'll have to host it myself
